texi2rst.py

```diff
@@ -268,10 +268,12 @@ def __init__(self):
             self.split_kinds = ('chapter', 'section', )
 
         def previsit_element(self, element):
-            if element.kind in self.split_kinds:
-                sectiontitle = element.first_element_named('sectiontitle')
-                if sectiontitle:
-                    text = sectiontitle.get_all_text()
+            # split '$target Option' subsections
+            sectiontitle = element.first_element_named('sectiontitle')
+            text = sectiontitle.get_all_text() if sectiontitle else None
+            if (element.kind in self.split_kinds or
+                (args and 'gcc.xml' in args.xml_file and element.kind == 'subsection' and text.endswith('Options'))):
+                if text:
```

Comment:
How about moving this condition out to a `should_split` method, to contain the special-casing there?

Reply:
Makes sense.